.github/pull_request_template.md

```diff
@@ -1,18 +1,19 @@
-Summary:
+**Summary**:
 
-Related issue: 
+**Related issue**: 
 
-Suggested reviewer(s):
+**Suggested reviewer(s)**:
 
-Checklist:
+**Checklist**:
 
 - [ ] The headers of all files contain a reference to the repository license (i.e., "License: This file is part of py-mmd-tools, licensed under the Apache License 2.0 (https://www.apache.org/licenses/LICENSE-2.0)")
 - [ ] The test coverage increased or remained the same as before
 - [ ] Every function is accompanied with a test suite
 - [ ] Tests are both positive (testing that the function work as intended with valid data) and negative (testing that the function behaves as expected with invalid data, e.g., that correct exceptions are thrown)
-- [ ] Functions with optional arguments has separate tests for all options
+- [ ] Functions with optional arguments have separate tests for all options
```

Comment:
I believe these 3 requirements are already included in the requirement for a good test coverage, as you can't have good coverage without them. Maybe better to add them as ¨sub-requirements¨ of the coverage requirement?

Reply:
yes, they are specifications but still useful in the checklist, I think. 

So like this(?):

- [ ] 100% test coverage
  - [ ] Every function is accompanied with a test suite
  - [ ] Tests are both positive (testing that the function work as intended with valid data) and negative (testing that the function behaves as expected with invalid data, e.g., that correct exceptions are thrown)
  - [ ] Functions with optional arguments have separate tests for all options

I can update that.